src/components/meeting/Meeting.js

```diff
@@ -3,23 +3,32 @@ import {useParams} from "react-router-dom";
 import {interviewAPI} from "../../utils/apis/InterviewAPI";
 import {MeetingDisplayName} from "./MeetingDisplayName";
 import {JitsiWrapper} from "./JitsiComponentWrapper";
+import {CircularProgress} from "@material-ui/core";
 
 export const Meeting = () => {
     let { interviewId, companyId } = useParams()
     const [name, setName] = useState(null)
+    const [isFetching, setIsFetching] = useState(true)
 
     useEffect(() => {
         if(companyId === undefined) {
             interviewAPI.getJobSeekerNameByInterviewId(interviewId)
-                .then(data => setName(data))
+                .then(data => {
+                    setName(data?.name)
+                    setIsFetching(false)
+                })
+        } else {
+            setIsFetching(false)
         }
     }, [companyId, interviewId])
 
     return (
         <div>
-            {name === null ?
+            {
+                isFetching ? <CircularProgress /> :
+                name === null ?
```

Comment:
Nie jest jakoś tak w konwencji jsowej, że jak się porównuje do nulla to się robi == zamiast ===?

Reply:
Wgl ciężko się czyta ten fragment, wyrzuciłbym ten cały drugi warunek logiczny do osobnego komponentu (w sensie komponentu funkcyjnego wewnątrz tego komponentu)